DEVELOPMENT.md

```diff
@@ -59,10 +59,18 @@ $ cd extra_tests
 $ pytest -v
 ```
 
-Rust unit tests can be run with `cargo`:
+Rust unit tests can be run by compiling RustPython and running the test module:
 
 ```shell
-$ cargo test --all
+$ cargo run --release -- -m test
```

Comment:
it might also be worth mentioning the `-j` flag for parallel execution. It usually works perfectly fine and saves you a couple of minutes of waiting.

Reply:
Good idea! Changed to include it.